rpc/test/helpers.go

```diff
@@ -141,7 +141,9 @@ func StartTendermint(app abci.Application, opts ...func(*Options)) *nm.Node {
 // StopTendermint stops a test tendermint server, waits until it's stopped and
 // cleans up test/config files.
 func StopTendermint(node *nm.Node) {
-	node.Stop()
+	if err := node.Stop(); err != nil {
+		fmt.Print("node is already stopped")
```

Comment:
???

Reply:
add it as a log so we can at least see the error